### DISCUSSION TITLE
Textured 2d polygon

### DISCUSSION THREAD

**Discussion by Ron5s** [Kode/Kha#1329] [General] [answered]
Hello,

Does Kha have a function to texture a 2d polygon, I haven't been able to find one.  if not what would be the best way to do that?  Use graphics4 and an orthographic camera to fake 2d?

Thanks for the help,
Ron

**Comment by RobDangerous** [comment, marked as answer]
G4 is all about textured polygons of any kind and you don't even need to use a projection.

**Reply by Ron5s** [reply]
That’s wonderful. I’ll look through G4 and figure it out. Thank you.
